src/Infrastructure/Command/CommandModule.php

```diff
@@ -24,17 +24,21 @@ public function services(): array
                 $container->get(DegreeProgramCollectionRepository::class),
                 $container->get(DegreeProgramRepository::class),
             ),
+            DegreeProgramSlugUpdater::class => static fn(ContainerInterface $container) => new DegreeProgramSlugUpdater(
+                $container->get(DegreeProgramBulkUpdater::class),
+            ),
             RelatedPostMetaRemover::class => static fn() => new RelatedPostMetaRemover(),
         ];
     }
 
     public function run(ContainerInterface $container): bool
     {
         add_action(
-            'edited_' . DegreeTaxonomy::KEY,
-            [$container->get(DegreeProgramBulkUpdater::class), 'whenDegreeTermUpdated']
+            'updated_term_meta',
+            [$container->get(DegreeProgramSlugUpdater::class), 'whenTermMetaUpdated'],
+            10,
+            3
         );
```

Comment:
If there's no specific need to remove that line, let's restore it for better readability. 
```suggestion
        );

```